frontend/fingertips-frontend/components/molecules/Inequalities/Table/index.tsx

```diff
@@ -0,0 +1,134 @@
+import { Table } from 'govuk-react';
+import styled from 'styled-components';
+import {
+  HealthDataForArea,
+  HealthDataPoint,
+} from '@/generated-sources/ft-api-client';
+import {
+  getDisplayedValue,
+  StyledAlignLeftHeader,
+  StyledAlignLeftTableCell,
+  StyledAlignRightHeader,
+  StyledAlignRightTableCell,
+  StyledDiv,
+  StyledTableCellHeader,
+} from '@/lib/tableHelpers';
+import {
+  getDynamicKeys,
+  getYearDataGroupedByInequalities,
+  groupHealthDataByYear,
+} from '@/components/organisms/Inequalities/inequalitiesHelpers';
+import { ReactNode } from 'react';
+
+export enum InequalitiesSexTableHeadingsEnum {
+  PERIOD = 'Period',
+}
+
+export interface InequalitiesSexTableRowData {
+  [key: string]: number | undefined;
+}
```

Comment:
This type isn't specific to representing sex, so should probably just be `InequalitiesTableRowData`
```suggestion
export interface InequalitiesTableRowData {
  [key: string]: number | undefined;
}
```